docs/components/buttons/Example.vue

```diff
@@ -3,13 +3,13 @@
 </script>
 
 <template>
-    <div class="space-x-8">
-        <w-button primary>Click me</w-button>
-        <w-button primary loading>Click me</w-button>
-        <w-button>Click me</w-button>
-        <w-button secondary quiet>Click me</w-button>
-        <w-button secondary quiet small>Click me</w-button>
-        <w-button primary negative>Click me</w-button>
-        <w-button negative quiet>Click me</w-button>
+    <div class="space-x-8 space-y-16 text-center">
```

Comment:
For some reason setting `space-y-8` or `space-x-16` did not work here. Testing it in drive gave correct css, so I'm not sure what's the matter.